pegasus/emails/2017_teacher_application_accept_teachercon.md

```diff
@@ -4,7 +4,7 @@ subject: "Congratulations from Code.org! Respond by April 21st"
 ---
 Hi <%= preferred_first_name_s %>,
 
-We are so excited to offer you a space in Code.org’s Professional Learning Program for <%= course_name_s %>! We think you can make an important contribution to our collective goal to ensure computer science education is available to every student in every school across the country, and we can’t wait to start working with you. Please read through the following information, and then make sure to let us know your plans via the form below.
+Code.org and <%= regional_partner_name_s %> are so excited to offer you a space in Code.org’s Professional Learning Program for <%= course_name_s %>! We think you can make an important contribution to our collective goal to ensure computer science education is available to every student in every school across the country, and we can’t wait to start working with you. Please read through the following information, and then make sure to let us know your plans via the form below.
```

Comment:
I added a check when generating the csv, which will raise an error if it's missing (and associated unit test)